_data/reference.yml

```diff
@@ -1,73 +1,88 @@
 bigheader: "Reference Documentation"
 abstract: "Design docs, concept definitions, and references for APIs and CLIs."
 toc:
-- docs/reference.md
+- docs/reference/index.md
 
-- title: "Kubernetes Resource Types"
+- title: "API Reference"
   section:
   - title: Version 1.6
-    path: /docs/resources-reference/v1.6/
+    section:
+    - title: API
+      path: /docs/api-reference/v1.6/
+    - title: Resource Types
+      path: /docs/resources-reference/v1.6/
   - title: Version 1.5
-    path: /docs/resources-reference/v1.5/
+    section:
+    - title: API
+      path: /docs/api-reference/v1.5/
+    - title: Resource Types
+      path: /docs/resources-reference/v1.5/
+  - title: Federation
+    section:
+      - title: federation/v1beta1
+        section:
+        - docs/reference/federation/v1beta1/operations.html
+        - docs/reference/federation/v1beta1/definitions.html
+      - title: v1
+        section:
+        - docs/reference/federation/v1/operations.html
+        - docs/reference/federation/v1/definitions.html
+      - title: extensions/v1beta1
+        section:
+        - docs/reference/federation/extensions/v1beta1/operations.html
+        - docs/reference/federation/extensions/v1beta1/definitions.html
+  - docs/reference/extensions/v1beta1/definitions.html
+  - docs/reference/labels-annotations-taints.md
+  - title: OpenAPI and Swagger
+    section:
+    - title: OpenAPI Spec
+      path: https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/
+    - title: Swagger Spec
+      path: https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/
 
-- title: "Kubernetes API"
+- title: "Using the API"
   section:
-  - title: Overview
-    path: /docs/concepts/overview/kubernetes-api/
-  - title: Version 1.6
-    path: /docs/api-reference/v1.6/
-  - title: Version 1.5
-    path: /docs/api-reference/v1.5/
+  - docs/reference/api-overview.md
   - title: Accessing the API
     section:
     - docs/admin/accessing-the-api.md
     - docs/admin/authentication.md
     - docs/admin/bootstrap-tokens.md
-    - title: Authorization Plugins
-      section:
-      - docs/admin/authorization/index.md
-      - docs/admin/authorization/rbac.md
     - docs/admin/admission-controllers.md
     - docs/admin/service-accounts-admin.md
-  - docs/api-reference/labels-annotations-taints.md
-  - docs/api-reference/extensions/v1beta1/definitions.html
+  - title: Authorization Plugins
+    section:
+    - docs/admin/authorization/index.md
+    - docs/admin/authorization/rbac.md
 
-- title: kubectl CLI
+- title: Working with Kubernetes using kubectl
```

Comment:
I am still reluctant to call it `Working with Kubernetes using kubectl`.

If I'm reading reference docs I already know one uses kubectl to "work with kubernetes". So this title sounds a bit obsolete to me.

It also feels like tutorials & tasks articles such as the following should be listed here if the title is `Working with Kubernetes using kubectl`:

-  Managing Kubernetes Objects Using Imperative Commands
- Installing and Setting Up kubectl
- Listing All Container Images Running in a Cluster

but clearly they don't belong here.